### ISSUE TITLE
Thank you XIAOMI! for releasing unbuildable kernel source!

### ISSUE THREAD

**Issue by Aaaaaaziskurnia** [MiCode/Xiaomi_Kernel_OpenSource#2205]
It's such a waste time to me for downloading and trying to build but never make a move. try changing base to latest caf and importing all your changes to drivers and devices tree manually by comparing file make me sick and still not buildable! JUST DELETE YOUR SHITTY REPOSITORY! I've been have nice thing with other manufactories even they is not uploading to github to delete their history of changing kernel just a single file of taball, they are still buidable and making the development goes fast!

**Comment by SsNiPeR1** [comment]
Yes, I can't DOWNLOAD it, not compile, compiling is easy, but downloading from branch?
